virtual_rainforest/animals/dummy_animal_module.py

```diff
@@ -0,0 +1,93 @@
+"""The 'dummy_animal_module.
+
+This file provides toy animal module functionality as well as self-contained
+dummy versions of the abiotic, soil, and plant modules that are required for
+setting up and testing the early stages of the animal module.
+"""
+
+# experimental file for figuring out how to make an animal module
+
+# to do
+# - rework dispersal
+# - send portion of dead to carcass pool
+
+# current simplifications
+# - only herbivory (want: carnivory and omnivory)
+# - only endothermy (want: ectothermy)
+# - only iteroparity (want: semelparity)
+# - no development
+
+# notes to self
+# assume each grid = 1 km2
+# assume each tick = 1 day (28800s)
+# damuth ~ 4.23*mass**(-3/4) indiv / km2
+# waste_energy pool likely unnecessary
+#   better to excrete directly to external pools
+# only elephants disperse atm
+
+
+class Plant:
+    """This is a dummy class of plant cohorts for testing the animal module."""
+
+    def __init__(self, name: str, mass: float, position: int) -> None:
+        """The constructor for Plant class."""
+        self.name = name
+        """The functional-type name of the plant cohort."""
+        self.mass = mass
+        """The mass of the plant cohort [g]."""
+        self.energy = mass * 100
+        """The amount of energy in the plant cohort [j] [toy]."""
+        self.alive = "alive"
+        """Whether the cohort is alive [True] or dead [False]."""
+        self.energy_max = mass * 100
+        """The maximum amount of energy that the cohort can have [j] [toy]."""
+        self.position = position
+        """The grid location of the cohort [0-8]."""
+
+    def plant_growth(self) -> None:
```

Comment:
A good rule of thumb is to start your function names with a verb, because they should be doing something. Given that it's a member of the `Plant` class, the `plant_` prefix is redundant too. I'd go for something like `grow()` or `increase_energy()`.